index.html.in

```diff
@@ -38,6 +38,7 @@
             <script src="//oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
             <script src="//oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
         <![endif]-->
+        <script type="text/javascript" src="./js/init.js"></script>
```

Comment:
Why was this added separately? Pretty certain it shouldn't be here.

Reply:
Sorry, needed a moment to figure out how things get loaded ...